### PR TITLE
Fix #50 - Broken docs

### LINKED ISSUE #50: Broken docs
Whenever I navigate to our [docs](https://humphd.github.io/next/), it does not load images.

docs_broken

### PR DESCRIPTION
**Pull request by mmBabol**
Fixed broken doc issue from PR #35, with the CSS and images not showing up. 

In the `siteConfig.js` file:
 * fixed the baseUrl to `https://humphd.github.io/next/`
 * corrected favicon path to point to the `img/favicon/` directory

Added missing files/directories into the `.gitignore` file for Docusaurus

Removed obsolete values from the `en.json` file. The doc files have now been renames and we do not need the old values in here anymore.

Added missing `database.svg` image.

New doc page should load correctly with the proper css & images as shown below

working home page

** Update **

* Removed ```src/editor/.DS_Store``` from ```.gitignore```
* To fix the auto-generated values in the ```en.json``` file, I had to fix the case inconsistency in all of the doc files. The generate ```en.json``` file is now generated correctly 
* Minified the SVG image file size

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -60,6 +60,20 @@ typings/
 # Docusaurus build folder
 website/build/
 
+.DS_Store
```
Comment (humphd) [inline .gitignore]: You can probably get rid of:

```
#OSX .DS_Store files
 src/editor/.DS_Store
```

File: website/i18n/en.json [hunk 2]
```diff
@@ -9,18 +9,12 @@
     "community": "Community",
     "Community": "Community",
     "introduction": "Introduction",
-    "Next introduction": "Next introduction",
```
Comment (humphd) [inline website/i18n/en.json]: Let's do these changes in the src file vs. this one, which is generated.

File: website/i18n/en.json [hunk 3]
```diff
@@ -9,18 +9,12 @@
...
-    "doc2": "document number 2",
-    "doc3": "This is document number 3",
-    "doc4": "Other Document",
-    "doc5": "Fifth Document",
     "Introduction": "Introduction",
     "technologies": "Technologies",
```
Comment (humphd) [inline website/i18n/en.json]: Let's not double include lowercase and Uppercase versions of strings.

File: .gitignore [hunk 4]
```diff
@@ -60,6 +60,21 @@ typings/
 # Docusaurus build folder
 website/build/
 
+#All OSX .DS_Store files
+.DS_Store
+
+lib/core/metadata.js
+lib/core/MetadataBlog.js
```
Comment (humphd) [inline .gitignore]: These can go

File: .gitignore [hunk 5]
```diff
@@ -60,6 +60,21 @@ typings/
...
+
+# Docusaurus website directory contents
+
+website/translated_docs
+website/build/
+website/yarn.lock
+website/node_modules
+website/i18n/*
+!website/i18n/en.json
```
Comment (humphd) [inline .gitignore]: Let's remove this, and also do `git rm -f website/i18n/en.json`

File: website/siteConfig.js [hunk 6]
```diff
@@ -14,7 +14,7 @@ const users = [
     caption: 'User1',
```
Comment (humphd) [inline website/siteConfig.js]: Let's get rid of the whole `users` array and see if it works, since we don't have other projects involved with ours yet!

**Comment by mmBabol** [comment]
I fixed the issues that we discussed today:
* Removed duplicate ```.DS_Store``` variable from ```.gitignore``` and reorganized the docusaurus gitignore section
* Added ```en.json``` to ```.gitignore```
* Removed the ```en.json``` file
* Fixed capitalization inconsistencies in ```siteConfig.js``` for ```doc: 'Introduction'``` & ```doc: 'Community'```

**Pull request merged**